src/modules/new-like.js

```diff
@@ -0,0 +1,30 @@
+async function newLike(id) {
```

Comment:
As per the project requirements, you should be using ES6 features, that include the use of arrow functions instead of the old function declaration way. Kindly update your code accordingly.